python/katana/local/_graph.pyx

```diff
@@ -219,45 +219,49 @@ cdef class GraphBase:
 
         return pyarrow_unwrap_table(arrow_table)
 
-    def add_node_property(self, table=None, **kwargs):
+    def add_node_property(self, table=None, TxnContext txn_ctx=None, **kwargs):
```

Comment:
Let's make `txn_ctx` keyword only. I think it should be since the normal usage will never pass it. That will make this declaration:

```python
def add_node_property(self, table=None, *, TxnContext txn_ctx=None, **kwargs):
```

Note the bare star  between the normal arguments and the keyword-only arguments.